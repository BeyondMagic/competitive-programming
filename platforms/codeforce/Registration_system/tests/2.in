15
first
first
second
second
third
third
third
third
third
third
third
third
third
third
third
