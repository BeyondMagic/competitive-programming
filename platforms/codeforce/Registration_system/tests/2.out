OK
first1
OK
second1
OK
third1
third2
third3
third4
third5
third6
third7
third8
third9
third10
